packages/box/src/Box.tsx

```diff
@@ -1,27 +1,21 @@
-import type { ReactNode } from 'react';
+import type { HTMLAttributes } from 'react';
 import type { Sprinkles } from './styles/rainbow-sprinkles.css';
 
 import { Slot } from '@radix-ui/react-slot';
+import { cx } from 'class-variance-authority';
 
 import { rainbowSprinkles } from './styles/rainbow-sprinkles.css';
 
-type BoxProps = Sprinkles & {
-	children?: ReactNode;
+interface BoxProps extends Sprinkles, Omit<HTMLAttributes<HTMLDivElement>, 'color'> {
 	asChild?: boolean;
-	'data-test-id'?: string;
-};
+}
 
-const Box = ({
-	asChild,
-	children,
-	'data-test-id': testId = asChild ? undefined : 'box',
-	...props
-}: BoxProps) => {
+const Box = ({ asChild, children, className: classes, ...props }: BoxProps) => {
 	const Component = asChild ? Slot : 'div';
 	const { className, style, otherProps } = rainbowSprinkles(props);
 
 	return (
-		<Component className={className} style={style} data-test-id={testId} {...otherProps}>
+		<Component className={cx(classes, className)} style={style} {...otherProps}>
```

Comment:
Personally I find this a little confusing and would prefer to rename the sprinkles variable to make it more obvious what's happening, but that's minor and you can take or leave that suggestion.